### ISSUE TITLE
openshift oauth admin user is not admin in argocd

### ISSUE THREAD

**Issue by rbo** [cloud-native-robotz-hackathon/infrastructure#24]

**Comment by rbo** [comment]
I tried a new prov. with:

```
ocp4_workload_openshift_gitops_update_route_tls: true
ocp4_workload_openshift_gitops_rbac_update: true
ocp4_workload_openshift_gitops_rbac_policy: |
  g, {{ ocp4_workload_authentication_admin_user }}, role:admin
ocp4_workload_openshift_gitops_rbac_scopes: '[name,groups]'
ocp4_workload_openshift_gitops_setup_cluster_admin: true
```

But the problem still exist.

**Comment by rbo** [comment]
Fixed with update to gitops-1.15: https://github.com/rhpds/agnosticv/pull/18813
